### ISSUE TITLE
Mathjax and Youtube don't embed when hosting via Google Drive.

### ISSUE THREAD

**Issue by ianelletson** [Dynalon/mdwiki#196]
For any LaTeX I get the raw code and for youtube I just get a link.
I confirmed that this is a problem at least on Google Drive because I opened the local file in my browser and it worked fine.

**Comment by mccullerlp** [comment]
Agreed, just saw this issue using with gitlab. Firefox and many other browsers now dissallow "mixed active content" Mathjax will fail to load anywhere it is hosted from a secure connection.

Your fix also worked for me.
